Vagrantfile

```diff
@@ -1,33 +1,70 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+require 'yaml'
+
+ansible_vars = YAML.load_file('provisioning/vars.yml')
+
+app_name = ansible_vars["nodejs_app_name"]
+
+app_directory = ansible_vars["nodejs_app_install_dir"]
+
+# Check for the existence of 'VM_HOST_TCP_PORT' or 'VM_GUEST_TCP_PORT'
+# environment variables. Otherwise if 'nodejs_app_tcp_port' is defined
+# in vars.yml then use that port. Failing that use defaults provided
+# in this file.
+host_tcp_port = ENV["VM_HOST_TCP_PORT"] || ansible_vars["nodejs_app_tcp_port"] || 8080
```

Comment:
@amb26  I changed the default port to 8088. I'm not sure I understand what the issue is with the environment variables though. Although I may be misunderstanding how they are all used. For dev use you'd just change `nodes_app_tcp_port` in vars.yml. If you weren't using the VM you could set the `FIRST_DISCOVERY_SERVER_TCP_PORT` environment variable instead.